content/resources/help-centre/software-tips/path.md

```diff
@@ -0,0 +1,96 @@
+---
+title: The PATH variable
+---
+
+## What is the "PATH"?
+
+If you have been following the lessons and help guides available on this
+website, and installing software such as FFmpeg and AP (AnalysisPrograms) then
+you have probably seen the term `PATH` by now. 
+
+The `PATH` variable is a system wide variable (known as en environment variable)
+that any program can access. The `PATH` variable contains a list of folders that
+your computer can check when it searches for programs to run.
+
+In simple terms, if we want to use a command from a particular software in our
+shell (see the [command line]({{% ref "command-line.md" %}}) guide), we have two
+choices. We can specify the full path to the `.exe` file, and then write our
+code:
+
+```ps
+> C:\Users\Anthony\AP\AnalysisPrograms.exe 
+```
+
+But this would be annoying to write out every time. Instead, if our `PATH`
+variable includes the AP folder, we can run AP commands without specifying this
+full path! 
+
+```ps
+> AnalysisPrograms.exe
+```
+
+## How to check the PATH variable
+
+We can quickly check if a program has been successfully added to our `PATH` or
+not. Open up a shell (such as PowerShell on Windows, or terminal on
+Linux/macOS), and type a command for the program we are interested in testing. 
+
+```ps
+> FFmpeg
+```
+
+If the command successfully returns some kind of output about the package, then
+it is available on your `PATH`.  
+
```

Comment:
you probably want to show them how to see path, 

`printenv | grep PATH` in bash and ` echo $env:Path` in pwsh